services/booking/search.rb

```diff
@@ -0,0 +1,22 @@
+class BookingService
+  class Search < ApplicationService
+    def initialize(params)
+      @route_id = params[:route_id]
+    end
+
+    def call
+      {
+        first_class:  orders.map(&:first_class_seats_count).sum,
+        second_class: orders.map(&:second_class_seats_count).sum
```

Comment:
Just an FYI - there's an alternative using `inject`
https://stackoverflow.com/questions/1538789/how-to-sum-array-of-numbers-in-ruby
You can benchmark it using `Benchmark`module to determine which one is faster
http://rubylearning.com/blog/2013/06/19/how-do-i-benchmark-ruby-code/